drivers/volume/portworx/portworx.go

```diff
@@ -61,6 +61,9 @@ const (
 	clusterPort                 = "port"
 	remoteKubeConfigPath        = "/tmp/kubeconfig"
 	pxMinVersionForStorkUpgrade = "2.1"
+	pxctlLocalSnapshotCreate 	= "pxctl volume snapshot create %s --name %s"
```

Comment:
nit: ident may fail fmt check, and feel it's weird to use %s this way..

Reply:
I've seen format strings used like this before, though I'd prefer to see `Format` in the name of the constant. Also yeah, funky indents